examples/reliable_conversation/mcp_agent.config.yaml

```
@@ -0,0 +1,43 @@
+  # Quality control settings
+  quality_threshold: 0.8
+  max_refinement_attempts: 3
+  consolidation_interval: 3
+  evaluator_model_provider: "openai" # or anthropic
+  
+  # Display and UX settings
+  verbosity: "normal"  # minimal, normal, verbose
+  show_internal_messages: true  # Show LLM interactions and workflow steps
+  verbose_metrics: false  # Show detailed quality metrics after each response
+  show_timing: false  # Show execution timing information
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)